requirements/dev-requirements.in

```diff
@@ -0,0 +1,22 @@
+-r requirements.txt
```

Comment:
this was originally `test-requirements.txt`, and I think we should set it back to that.